files/jsons/Policies/appID_Management/ptu_1.json

```diff
@@ -0,0 +1,1277 @@
+{
+    "policy_table": {        

```

Comment:
@dboltovskyi  Updated policy table should contain sections: module_config, functional_groupings, consumer_friendly_messages, and app_policies.
See https://adc.luxoft.com/confluence/display/APPLINK/Policies+-+appID+Management
 AppLink Policies Manager Specification v1.3.doc